frame/contracts/src/wasm/runtime.rs

```diff
@@ -813,6 +818,24 @@ define_env!(Env, <E: Ext>,
 		ctx.set_storage(key_ptr, value_ptr, value_len).map(|_| ())
 	},
 
+	// Replace contract code
+	//
+	// # Parameters
+	//
+	// - code_hash_ptr: a pointer to the buffer that contains new contract code
+	//
+	// # Errors
+	//
+	// - requested buffer is not within the bounds of the sandbox memory.
+	// - the buffer contents cannot be decoded as the required type.
```

Comment:
Those are implicit. We don't document them here. Also, those aren't errors but panics.